R/contact_matrix.R

```diff
@@ -783,7 +794,7 @@ contact_matrix <- function(
         c(
           "{.code symmetric = TRUE} does not work with missing data; will \\
           not make matrix symmetric.",
-          "i" = "{warning.suggestion}"
+          "i" = "{warning.suggestion}" # nolint
```

Comment:
```suggestion
          # nolint start
          "i" = "{warning.suggestion}"
          # nolint end
```
unless there is a reason for the different format here?